test/unit/math/prim/scal/prob/neg_binomial_2_test.cpp

```diff
@@ -202,3 +202,14 @@ TEST(ProbDistributionsNegBinomial, chiSquareGoodnessFitTest4) {
 
   EXPECT_TRUE(chi < quantile(complement(mydist, 1e-6)));
 }
+
```

Comment:
There should be a derivative test, too, in rev.  I'm not sure how to plug this into the testing framework, but @syclik will.